docs/connector-development/connector-builder-ui/tutorial.md

```diff
@@ -0,0 +1,202 @@
+
+When used in a connection, the connector will make sure exchange rates for the same day are not requested multiple times - the date of the last fetched record will be stored and the next scheduled sync will pick up from where the previous one stopped.
+
+## Adding transformations
+
+Note that a warning icon should show next to the "Detected schema" tab - using the per-date endpoint instead of the latest endpoint slightly changed the shape of the records by adding a `historical` property. As we don't need this property in our destination, we can remove it using a transformation.
+
+To do so, follow these steps:
+* Enable the "Transformations" section
+* Add a new transformation
+* Set the "Path" to `historical`
+* Trigger a new test read
+
+The `historical` property in the records tab and the schema warning should disappear.
+
+## Publishing and syncing
+
+So far, the connector is only configured as part of the connector builder project. To make it possible to use it in actual connections, you need to publish the connector. This captures the current state of the configuration and makes the connector available as a custom connector within the current Airbyte workspace.
+
+To use the connector for a proper sync, follow these steps:
+* Click the "Publish" button and publish the first version of the "Stock Exchange" connector
+* Go to the "Connections" page and create a new connection
+* As Source type, pick the "Stock Exchange" connector you just created
+* Set API Key, base currency and start date for the sync - to avoid a large number of requests, set the start date to one week in the past
+* Click "Set up source" and wait for the connection check to validate the provided configuration is valid
+* Set up a destination - to keep things simple let's choose the "E2E Testing" destination type
+* Click "Set up destination", keeping the default configurations
+* Wait for Airbyte to check the record schema, then click "Set up connection" - this will create the connection and kick off the first sync
+* After a short while, the sync should complete successfully
+
+Congratulations! You just completed the following steps:
+* Configured a production-ready connector to extract currency exchange data from an HTTP-based API:
+  * Configurable API key, start date and base currency
+  * Incremental sync to keep the number of requests small
+  * Schema declaration to enable normalization in the destination
+* Tested whether the connector works correctly in the UI
+* Made the working connector available to configure sources in the workspace
+* Set up a connection using the published connector and synced data from the Exchange Rates API
+
+## Next steps
+
+This tutorial didn't go into depth about all features that can be used in the connector builder. Check out the concept pages for more information about certain topics:
+* Authentication
```

Comment:
nit: let's add links to these

Reply:
yeah, I will do that once everything is merged together (can't point to non-existing pages as there's a link checker testing these)